src/utils/build.js

```diff
@@ -110,13 +111,21 @@ const listFiles = (dir) => {
   });
 };
 
-const forceIncludeDumbPath = (filePath/*, smartPaths*/) => {
-  // we include smartPaths just incase you want to check the inclusion of some library
+const forceIncludeDumbPath = (appConfig, filePath) => {
+  let matchesConfigInclude = false;
+  const configIncludePaths = _.get(appConfig, 'includeInBuild', []);
+  _.each(configIncludePaths, (includePath) => {
+    if (filePath.match(RegExp(includePath, 'i'))) {
+      matchesConfigInclude = true;
```

Comment:
Should probably to add `return false;` here to break the loop.

Reply:
Nice catch! I didn't do it initially because this should never be a long list, but there's no need to unnecessarily run through the loop anyway.

Reply:
Unfortunately, turns out `return false;` there isn't stopping the iterator. You can test with:

```js
_.each(['a', 'b', 'c'], (letter) => {
    console.log(letter);
    if (letter === 'b') {
      return false;
    }
  });
```

Reply:
Code updated!